install/arch/InstallHalyard.sh

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env bash
+
+set -e
+
+function process_args() {
+  while [ "$#" -gt "0" ]
+  do
+    local key="$1"
+    shift
+    case $key in
+      --halyard-bucket-base-url)
+        echo "halyard-bucket-base-url"
+        HALYARD_BUCKET_BASE_URL="$1"
+        shift
+        ;;
+      --download-with-gsutil)
+        echo "download-with-gsutil"
+        DOWNLOAD_WITH_GSUTIL=true
+        ;;
+      --spinnaker-repository)
+        echo "spinnaker-repo"
+        SPINNAKER_REPOSITORY_URL="$1"
+        shift
+        ;;
+      --spinnaker-registry)
+        echo "spinnaker-registry"
+        SPINNAKER_DOCKER_REGISTRY="$1"
+        shift
+        ;;
+      --spinnaker-gce-project)
+        echo "spinnaker-gce-project"
+        SPINNAKER_GCE_PROJECT="$1"
+        shift
+        ;;
+      --config-bucket)
+        echo "config-bucket"
+        CONFIG_BUCKET="$1"
+        shift
+        ;;
+      --user)
+        echo "user"
+        HAL_USER="$1"
+        shift
+        ;;
+      --version)
+        echo "version"
+        HALYARD_VERSION="$1"
+        shift
+        ;;
+      -y)
+        echo "non-interactive"
+        YES=true
+        ;;
+      --help|-help|-h)
+        print_usage
+        exit 13
+        ;;
+      *)
+        echo "ERROR: Unknown argument '$key'"
+        exit -1
+    esac
+  done
+}
+
+function get_user() {
+  local user
+
+  user=$(who | awk '{print $1;}')
```

Comment:
Won't moving from `who -m` to `who` mean we'll print out all of the users currently logged in and store them in `user`?  I wonder if this can just be replaced by `user=$(whoami)` (which should also work for Debian, reducing the differences between the install scripts).

Reply:
whoami works, you are right. I will do some work on this this week.